local_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Create a hosts entry for findthemasks
+# So Google API key can work
+if ! grep 'localhost.findthemasks.com' /etc/hosts; then
+  echo "Creating hosts entry for local.findthemasks.com"
+  sudo echo '127.0.0.1 localhost.findthemasks.com local.findthemasks.com' | sudo tee -a /etc/hosts
```

Comment:
is this required? local.findthemasks.com:8080 has been working for me without any /etc/hosts changes

Reply:
Oh nice I didn't realize this has a real DNS record.